qemu/tests/ksm_overcommit.py

```diff
@@ -449,7 +450,7 @@ def split_parallel():
     if (guest_reserve == -1):
         # default guest_reserve = minimal_system_mem(256MB)
         # later we add tmpfs overhead
-        guest_reserve = 256
+        guest_reserve = 1536
```

Comment:
Could you explain more about this ?  And the comment in L451 need be updated accordingly.

Reply:
256M is not enough now. We may hit OOM easily, which is not what we desire.  As far as I'm concerned, 1.5G is more safer.